libminifi/test/unit/PropertyTests.cpp

```diff
@@ -107,3 +107,28 @@ TEST_CASE("Test Trimmer Left", "[testTrims]") {
   REQUIRE(test.c_str()[1] == ' ');
 }
 
+TEST_CASE("Test int conversion", "[testStringToInt]") {
```

Comment:
Most of these would fail without this change.

Reply:
Almost all of these are invalid and shouldn't be accepted, though. This was the point of https://issues.apache.org/jira/browse/MINIFICPP-707. I don't want to trounce on this PR, but the reason I created the ticket was that some of these SHOULD be prevented from being used at all in configs, fail early. Therefore it's not as simple as doing what is done here. 710 should be OBE in favor of 707.

Reply:
I didn't think my last comment was entirely helpful so I went through and tried to parse out the parts that are appropriate and SHOULD be accepted. Hopefully I was more clear on what we can immediately and what we can't accept without more discussion. If we can parse this out then we can approve and merge the good parts.